src/ec/suite_b/ecdsa.rs

```diff
@@ -50,6 +66,12 @@ enum ECDSAVerificationAlgorithmID {
     ECDSA_P384_SHA384_FIXED,
 }
 
+impl PartialEq for ECDSAVerificationAlgorithm {
+    fn eq(&self, other: &Self) -> bool { self.id == other.id }
+}
```

Comment:
I think we should not implement PartialEq for this type. Instead we should implement PartialEq for `signature::VerificationAlgorithm` in some way. Maybe we need to make `signature::VerificationAlgorithm` a struct that can hold the id field first. WDYT?